code/modules/mob/living/carbon/life.dm

```diff
@@ -314,7 +314,10 @@
 /mob/living/carbon/handle_status_effects()
 	..()
 	if(staminaloss)
-		adjustStaminaLoss(-3)
+		if(IsKnockdown() || IsUnconscious())

```

Comment:
As far as I can see there is no tick proc() for either the knockdown or unconscious status effect.

That or I can't find where they are supposed to be.